src/routes/groups/groups.controller.js

```diff
@@ -0,0 +1,60 @@
+const groupsData = require('./../../../db/groups.data.json')
+const fs = require("fs");
+const { promisify } = require("util");
+
+const writeFile = promisify(fs.writeFile);
+
+const listGroup = (req, res) => {
+  return res.json(groupsData);
+};
+
+const postGroup = async (req, res) => {
+  const id = groupsData.groups.length + 1;
```

Comment:
What will happen if some groups are deleted and then a new one is made? Will the id still be unique?